ucrop/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,7 @@
+<resources>
+    <string name="ucrop_label_original">原始比例</string>
+    <string name="ucrop_label_edit_photo">裁剪</string>
```

Comment:
This translates as Crop, not edit photo. @bung87 , please have a look. We will be glad to merge your PR once corrected.

Reply:
I use same translation as tw, maybe u mean this should translates as 'edit', I also compare to english and japanese , they all means "edit", so tw version maybe wrong ?